src/stories/index.js

```diff
@@ -45,6 +45,14 @@ storiesOf('Footer', module)
       const campaign = 'sportrelief';
       return (<Footer copy={copy} source={source} campaign={campaign} />);
     }),
+  )
+  .add('Red Nose Day',
+    withInfo('rednoseday.com footer')(() => {
+      const copy = text('Copy', 'copyright 2018');
+      const source = 'https://www.comicrelief.com'; // fallback to comicrelief.com
+      const campaign = 'rednoseday';
```

Comment:
Do we need to use this text function on each of the const vars here too? Not 100% sure what it's actually doing, though

Reply:
We need to pass value to props but don't need repeat const ;)  I only copied entire block from previous ones. Agree we can definitely tidy them up :)